### ISSUE TITLE
zone.js@^0.6.25 requirement not met

### ISSUE THREAD

**Issue by genuinefafa** [marcj/angular2-localStorage#78]
1st I tried to install as I use to, then, I get an error in `import { WebStorageModule, LocalStorageService } from "angular2-localstorage";` as it `angular2-localstorage` was not found.

Installed `typings` as someone suggested.
Installed by using `npm install marcj/angular2-localstorage`

Now, a new issue,

```bash
$ npm install marcj/angular2-localstorage

> angular2-localstorage@0.4.0 postinstall /Users/fafa/Develop/benjamin/frontend/node_modules/angular2-localstorage
> typings install

typings WARN deprecated 11/21/2016: "registry:dt/node#6.0.0+20160921192128" is deprecated (updated, replaced or removed)

└── node (global)

frontend@0.0.0 /Users/fafa/Develop/benjamin/frontend
├── angular2-localstorage@0.4.0  (git://github.com/marcj/angular2-localstorage.git#85b25caca6eb969632e258fa5587e85e871b38b6)
└── UNMET PEER DEPENDENCY zone.js@0.7.7

npm WARN angular2-localstorage@0.4.0 requires a peer of zone.js@^0.6.25 but none was installed.
$ npm install
npm WARN angular2-localstorage@0.4.0 requires a peer of zone.js@^0.6.25 but none was installed.
```

I do have a `package.json` with
```javascript
  "dependencies": {
...
    "zone.js": "^0.7.2"
  },
```

Any ideas?

pst: i also did `npm cache clean` and `rm -rf dist node_modules` before `npm install` again.

**Comment by dogofpavlov** [comment]
I ran into a similar problem. 
When I first did the install with "npm install --save angular2-localstorage" I got an error on the install 

npm ERR! angular2-localstorage@0.4.0 postinstall: `typings install`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the angular2-localstorage@0.4.0 postinstall script 'typings install'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the angular2-localstorage package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     typings install
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs angular2-localstorage

I managed to fix this by doing "npm install typings -global"... and then retried the install and it worked. 

But then when I go to use it with:
import { WebStorageModel, LocalStorageService } from 'angular2-localstorage';


I get: " Cannot find module 'angular2-localstorage'."

**Comment by genuinefafa** [comment]
@dogofpavlov try `rm -rf dist node_modules` and then `npm install` my guess is that you will see my troubling results...

**Comment by dogofpavlov** [comment]
@genuinefafa yes you are right, i have the same issue as you. I have since started using "angular-2-local-storage" instead, as I really don't have time to wait for a fix .

**Comment by genuinefafa** [comment]
I'll check it out! Thanks!
